src/Websites/Websites/help/Publish-AzWebApp.md

```diff
@@ -0,0 +1,184 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Websites.dll-Help.xml
+Module Name: Az.Websites
+online version: https://docs.microsoft.com/en-us/powershell/module/az.websites/publish-azwebapp
+schema: 2.0.0
+---
+
+# Publish-AzWebApp
+
+## SYNOPSIS
+Deploys an Azure Web App from a ZIP, JAR, or WAR file using zipdeploy. 
+
+## SYNTAX
+
+### FromResourceName
+```
+Publish-AzWebApp -ArchivePath <String> [-AsJob] [-ResourceGroupName] <String> [-Name] <String>
+ [[-Slot] <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### FromWebApp
+```
+Publish-AzWebApp -ArchivePath <String> [-AsJob] [-WebApp] <PSSite> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Publish-AzWebApp** cmdlet uploads content to an existing Azure Web App. The content should be packaged in a ZIP file if using stacks such as .NET, Python, or Node, or a WAR or JAR file if using Java. The content should be pre-built and ready-to-run without any additional build steps during deployment. This cmdlet uses the Kudu zipdeploy and wardeploy features to deploy content. Refer to the Kudu wiki for details about how zipdeploy and wardeploy work, and how to properly package a web app for deployment. https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file and https://github.com/projectkudu/kudu/wiki/Deploying-WAR-files-using-wardeploy contain helpful details about zipdeploy and wardeploy.
```

Comment:
You should use aka.ms links for those 2 wiki entries. These md files get published to docs. And, if those links every become invalid, it is really painful to have to fix them in docs pages. So, aka.ms links are the best solution.